statsd/options.go

```diff
@@ -69,8 +69,8 @@ type Options struct {
 	// The magic value 0 will set the option to the optimal size for the transport
 	// protocol used when creating the client: 2048 for UDP and 512 for UDS.
 	SenderQueueSize int
-	// WriteTimeoutUDS is the timeout after which a UDS packet is dropped.
-	WriteTimeoutUDS time.Duration
+	// WriteTimeout is the timeout after which a UDS packet is dropped.
```

Comment:
nit: is this for both UDS and UDP? If so, we should remove the specificity.
```suggestion
	// WriteTimeout is the timeout after which a packet is dropped.
```